P5/Source/Guidelines/en/PH-PrimarySources.xml

```diff
@@ -11,8 +11,8 @@ $Id$
      type="div1" xml:id="PH" n="18">
    <head>Representation of Primary Sources</head>
 
-   <p>This chapter describes elements that may be used to represent primary source materials,
-      such as manuscripts or other written materials. Some of these specialized elements,
+   <p>This chapter described elements for the encoding of materials where issues such as layout,
```

Comment:
I liked "describe" better but in fact this is not standardized in the Guidelines.

Reply:
That's likely a type, I will fix it. "This chapter describes"